Makefile

```diff
@@ -458,7 +458,7 @@ uninstall:
 # Linters --------------------------------------------- {{{1
 #
 
-PYTHON_SCRIPTS := utils/hare-shutdown utils/hare-status utils/gen-uuid
+PYTHON_SCRIPTS := utils/hare-shutdown utils/hare-status utils/gen-uuid utils/hare-node utils/util.py
```

Comment:
[suggestion]: rename utils/util.py to utils/utils.py

Reply:
done.